### PR TITLE
feat: adds tailwindcss/forms

### PR DESCRIPTION
**Pull request by cdrage**
feat: adds tailwindcss/forms

### What does this PR do?

Adds https://github.com/tailwindlabs/tailwindcss-forms in order to
correctly apply styles to forms / checkboxes

### Screenshot/screencast of this PR

Form styles are now applying (previously, tailwind would ignore css in `class=` of forms).

See screenshot of stylistic change of checkboxes:

Screen Shot 2022-11-07 at 10 43 38 AM

### What issues does this PR fix or reference?

N/A

### How to test this PR?

`yarn watch`

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
DRAFT as I'm going through each section and re-applying the styles / editing them.

**Comment by benoitf** [comment]
It looks great but we would need to fix some styles before probably (as I think we have some styles that are not rendered today but they should not be there probably)

image

There is an extra border on the search widget (currently it's like a single item)

The color of the checkbox should probably match the color of the badges


Checkboxes are not rendering properly in settings
image

Extra border probably is everywhere
image

File: tailwind.config.cjs [hunk 1]
```diff
@@ -28,5 +28,5 @@ module.exports = {
       },
     },
   },
-  plugins: [],
+  plugins: [require('@tailwindcss/forms')],
```
Comment (benoitf) [inline tailwind.config.cjs:31]: I think here we should use import instead of require

**Comment by cdrage** [comment]
> It looks great but we would need to fix some styles before probably (as I think we have some styles that are not rendered today but they should not be there probably)
> 
> image
> 
> There is an extra border on the search widget (currently it's like a single item)
> 
> The color of the checkbox should probably match the color of the badges
> 
> Checkboxes are not rendering properly in settings image
> 
> Extra border probably is everywhere image

Yup. That's why it's draft as I've yet to change anything anywhere yet.

**Comment by cdrage** [comment]
> It looks great but we would need to fix some styles before probably (as I think we have some styles that are not rendered today but they should not be there probably)
> 
> image
> 
> There is an extra border on the search widget (currently it's like a single item)
> 
> The color of the checkbox should probably match the color of the badges
> 
> Checkboxes are not rendering properly in settings image
> 
> Extra border probably is everywhere image

Yup! I'll make those changes, that's why I set it as draft for now. Thanks for pointing out those sections!

**Comment by cdrage** [comment]
Changes *way* too many sections ATM so closing this now until we revisit styling the checkboxes (may just use pure .css instead)

**Pull request closed without merging**